src/services

```diff
@@ -31,6 +31,7 @@ p({"class":"foam.nanos.boot.NSpec", "name":"exportDriverRegistryDAO",     "lazy"
+p({"class":"foam.nanos.boot.NSpec", "name":"menuDAO2",                  "lazy":true,  "serve":true,  "serviceScript":"return new foam.dao.PMDAO(new foam.dao.JDAO(foam.nanos.menu.Menu.getOwnClassInfo(), \"menus\"));","client":"{\"of\":\"foam.nanos.menu.Menu\"}"})
```

Comment:
This should be named menuDAO so that the client will read it and the old one should be removed.